cloud_automation/gcf/bt_trigger.go

```diff
@@ -0,0 +1,136 @@
+package btcachegeneration
```

Comment:
Please add some comments on what this does, and importantly, when in the flow this gets triggered and by whom.